dstep/driver/Application.d

```diff
@@ -40,57 +40,33 @@ class Application
     public void run ()
     {
         import std.exception : enforce;
+        import std.range;
 
         enforce!DStepException(config.inputFiles.length > 0,
             "dstep: error: must supply at least one input file\n");
 
         enforceInputFilesExist(config);
 
-        // when only one input file is supplied, -o argument is
-        // interpreted as file path, otherwise as base directory path
-        bool singleFileInput = config.inputFiles.length == 1;
+        auto inputFiles = config.inputFiles;
+        auto outputFiles = makeOutputFiles(config);
```

Comment:
Looks like it would be better to perform this step after `enforceTranslationUnitsCompiled`. if `enforceTranslationUnitsCompiled` fails, then `makeOutputFiles` won't be run.

Reply:
Yes, that makes sense.

Reply:
Fixed.